Formula/parity.rb

```diff
@@ -5,7 +5,7 @@ class Parity < Formula
   head "https://github.com/thoughtbot/parity.git"
   sha256 "600e81753d6b5b765371fe135883bfdb92812e0dfd7cb6eb8078ab7439389617"
   url "https://github.com/thoughtbot/parity/archive/v3.0.0.tar.gz"
-  version "3.0.0"
+  version "3.0.0.1"
```

Comment:
I assume this means that the homebrew-specific version gets bumped to `3.0.0.1` and the parity version stays at `3.0.0` and this disconnect doesn't really matter? (ie, whenever we have a parity 3.0.1 we'll also bump homebrew formula to 3.0.1 and it will be back in sync and we can put this all behind us)

Reply:
@mjankowski that's correct. I couldn't think of a better way to do this, we have to bump the Homebrew version number or I don't believe that users who already upgraded will get the missing change.

Reply:
Sounds good, let's see!